forge-gui/res/cardsfolder/t/tempting_wurm.txt

```diff
@@ -6,4 +6,6 @@ T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.S
 SVar:EachOpponent:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ TemptingChange
 SVar:TemptingChange:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Artifact,Creature,Enchantment,Land | DefinedPlayer$ Remembered | ChangeNum$ X
 SVar:X:Count$ValidHand Artifact.RememberedPlayerCtrl,Creature.RememberedPlayerCtrl,Enchantment.RememberedPlayerCtrl,Land.RememberedPlayerCtrl
+#NOTE: AI will play Tempting Wurm on turn two when the opponent still has a full hand.
+AI:RemoveDeck:All
```

Comment:
Better turned into a NeedsToPlay/NeedsToPlayVar hint so that this is not an issue.

Reply:
Yeah, I don't know how to program the AI logic; I'll let you do your magic here. ;)